src/app/shared/wysiwyg/wysiwyg.component.ts

```diff
@@ -24,6 +25,15 @@ import { PromptComponent } from './prompt.component';
         <button (click)="prompt.hide()">Cancel</button>
       </div>
     </publish-prompt>
+    <publish-prompt #prompt *ngIf="editor?.isSelectionEmpty()">
+      <h1 class="modal-header">Warning</h1>
+      <div class="modal-body" *ngIf="editor?.isSelectionEmpty()">
```

Comment:
any reason for the duplicated `*ngIf="editor?.isSelectionEmpty()"` here + on line 28?